ngmt/modules/ssd/_pham.py

```diff
@@ -132,7 +136,17 @@ def detect(
 
         # Select gyro data and convert it to numpy array format
         gyro = data.iloc[:, 3:6].copy()
-        gyro = gyro.to_numpy()
+        gyro = -gyro.to_numpy()
+
+        # Check unit of acceleration data if it is in g or m/s^2
+        if acceleration_data_unit == "m/s^2":
+            # Convert acceleration data from m/s^2 to g (if not already is in g)
+            accel /= 9.81
+
+        # Check unit of acceleration data if it is in g or m/s^2
```

Comment:
Update the comment for the gyro

Reply:
The comment for the gryo data was updated.